spring-boot-project/spring-boot/src/main/java/org/springframework/boot/env/EnvironmentPostProcessorApplicationListener.java

```diff
@@ -101,11 +101,11 @@ private void onApplicationEnvironmentPreparedEvent(ApplicationEnvironmentPrepare
 		}
 	}
 
-	private void onApplicationPreparedEvent(ApplicationPreparedEvent event) {
+	private void onApplicationPreparedEvent() {
 		finish();
 	}
 
-	private void onApplicationFailedEvent(ApplicationFailedEvent event) {
+	private void onApplicationFailedEvent() {
 		finish();
 	}
```

Comment:
It seems a bit ambiguous to call it a handler function for the event by deleting the event parameter. 
Wouldn't it be better to get rid of the wrapper function and call finish() right away?

Reply:
Thought about that actually, but I liked the clear separation of things here. For me they are still the containers for the logic that should happen in case of the respective event and just share the common `finish()` call - regardless of the fact that they don't need the parameter. But I have no strong feelings for either option. Let's see what the team thinks and I'll gladly change to whatever is preferred.

Reply:
I prefer it as currently proposed.

Reply:
Thanks for the considering.